src/cpu/testers/gpu_ruby_test/address_manager.cc

```diff
@@ -64,7 +64,9 @@ AddressManager::AddressManager(int n_atomic_locs, int n_normal_locs_per_atomic)
     std::shuffle(
         randAddressMap.begin(),
         randAddressMap.end(),
-        std::default_random_engine(random_mt.random<unsigned>(0,UINT_MAX))
+        // TODO: This is a bug unrelated to this draft PR but the GPU tester is
+        // useful for testing this PR.
+        std::default_random_engine(random_mt.random<unsigned>(0,UINT_MAX-1))
```

Comment:
maybe spacing off on this line?

Reply:
Do you want this removed?

Reply:
no ... it looks fine in this format but in the review format it looks like line 69 had incorrect spacing